spec/requests/api/v1/sign_in_spec.rb

```diff
@@ -0,0 +1,67 @@
+require 'rails_helper'
+require 'json'
+
+describe 'POST api/v1/users/sign_in', type: :request do
+  let(:params)       { FactoryBot.attributes_for(:user) }
+  let(:params_login) { { email: params[:email], password: params[:password] } }
+  before(:each, confirm_email: true) do
+    sign_up
+  end
+
+  context 'signs in successfully', confirm_email: true do
+    it 'returns status 200 OK' do
+      post user_session_path, params: params_login, as: :json
+      expect(response.status).to eq(200)
```

Comment:
Avoid using plain numbers here, you can use status code symbols (in this case `:ok`).